system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -631,6 +632,11 @@
 
 ;;; fixture - each test to call this fixture
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(defn set-collection-ingest-umm-version-to-current
```

Comment:
Rename this function to `set-ingest-umm-version-to-current` and include setting ingest UMM version of variables/services also to the most current version that CMR supports.

Reply:
Yeah, that's what I thought would be a good thing to do.  Done.

Reply:
Looks like all the variable/service tests passed with default variable-umm-version and service-umm-version set to 1.3 which doesn't exist.  This indicates that the tests don't use these default values for ingests.  They might have already used the latest umm versions.  Will write a ticket to investigate if it's true, if so, need to see if it's only for testing or if it's because ingest is actually using the latest umm version, rather than the accepted umm version.